backend/server.py

```diff
@@ -72,11 +75,29 @@ def handle_exception(e):
         return response
 
 
-def _get_content_from_site(url):
+def _get_content_from_site(url: str) -> str:
```

Comment:
Since we don't have header files in python, can we have docs on top of the API to describe its behavior as well as its parameters and return values?

Reply:
wait could you explain? I thought in python it was the convention to have docstrings at the top of each function, not at the top

Reply:
Yes I meant at the top of each function. I don't see a docstring here that's why I said it

Reply:
Ah so this function is going to be changed and documented in a further MR; can we leave it for now?

Reply:
Okay sounds good